bfillings/vsearch.py

```diff
@@ -0,0 +1,561 @@
+#!/usr/bin/env python
```

Comment:
Removed shebang from the library code, however why do I need to do this for the test code? Tests for Blast, bwa & other tools all have it.